src/bvh/hash.hpp

```diff
@@ -83,7 +83,11 @@ namespace bvh
 #ifdef __BMI2__
     inline std::uint64_t expand64intrin( std::uint64_t _21bit )
     {
+#if defined(KOKKOS_COMPILER_NVCC)
+      return expand64(_21bit);
+#else
       return _pdep_u64( _21bit, 0x1249249249249249u );
+#endif
```

Comment:
Non-intrinsic implementation is already available, so I'm just adding another `#ifdef` here.